### PR TITLE
Add SV browser

### PR DESCRIPTION
**Pull request by alexander-veit**
This PR adds the SV browser to CGAP.

Some other notable changes:
- All higlass packages received a new version, mostly to reduce the number of packages they depend on. 
- We no longer depend on external worker files. These have been removed from the build process (see `gulpfile.js`)

image

### CODE REVIEW

**Review by alexkb0009 — APPROVED** [review]
Code looks good! (haven't tested on local but don't anticipate any issues)

File: src/encoded/types/case.py [hunk 1]
```diff
@@ -307,6 +307,10 @@ def _build_case_embedded_list():
         # File linkTo
         "vcf_file.file_ingestion_status",
         "vcf_file.accession",
+
```
Comment (alexander-veit) [inline src/encoded/types/case.py:310]: Approved by Doug

**Review by willronchetti — APPROVED** [review]
LGTM

**Review by Bianca-Morris — APPROVED** [review]
Code looks good. 

Note: I haven't been able to get the browser working on my local, but it's probably a combination of inserts and maybe some other things specific to my env (BAM browser doesn't typically work for me either). I don't want to spend a whole bunch of time troubleshooting this (trying to finish some other tasks before I leave for vacation) or block merge on this if it's working for everyone else. Maybe after I get back we can try to figure out why it's not working, but for now I think if it's working for you all, it's probably fine to merge.

**Comment by alexander-veit** [comment]
Thank you for the reviews. I will merge it soon. The problem is most likely missing inserts, both for the BAM browser and the SV browser.

**Pull request merged**